### ISSUE TITLE
At least to me the networking documentation is hard to follow

### ISSUE THREAD

**Issue by rmohr** [kubevirt/user-guide#91]
>Can we follow a little bit more the structure like we have for https://kubevirt.io/user-guide/#/workloads/virtual-machines/disks-and-volumes? The structure there is lent from k8s, which I personally find pretty easy to follow and user-focused.
>
>So to have 
> * Examples per configs
> * For instance also show where and how to set the model
> * For a comprehensive and up-to-date list of supported values, link to the api-reference and make sure that the api-reference comment contains supported values.
>
> It may just be me, but I find the structure of this document very confusing and it is almost impossible for me to parse it and extract the necessary information. It also looks a little bit like this documentation tries to be a mixture of api-reference and user-guide. We should leverage the fact that we have an api-reference.

from https://github.com/kubevirt/user-guide/pull/89#issuecomment-399029970

**Comment by fabiand** [comment]
I totally agree. The networking do is structural wise not so good.

Technically the problem s that we have three components: frontend, connection, backend.

This makes the current situation so bad.

But we can probably flatten this by mentioning the relevant connection times in the context of frontend, then we again only have frontends and backends, which should allow us to follow the disk pattern.

My 2ct

**Comment by booxter** [comment]
@rmohr Agreed. About examples, how do we guarantee they are valid / still working as we move forward? Maybe we could consume them from kubevirt repo (cluster/examples/?) and allow validations to happen against each of examples. What do you think?

**Comment by fabiand** [comment]
> Maybe we could consume them from kubevirt repo (cluster/examples/?) and allow validations to happen against each of examples. What do you think?

Technically this works with docsify.

An exampe is here:
https://github.com/kubevirt/user-guide/pull/92

Gotot th evirtual machines section to see bar and an inline external yaml file.

**Comment by rmohr** [comment]
> @rmohr Agreed. About examples, how do we guarantee they are valid / still working as we move forward? Maybe we could consume them from kubevirt repo (cluster/examples/?) and allow validations to happen against each of examples. What do you think?

Anything that helps to keep them valid is great. I did not have a clear picture in my mind on how to do that. If you have ideas go ahead! :+1:

**Comment by booxter** [comment]
Please assign me to the issue (or give me access to the action). I will look into this during / after macAddress work.
